main.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"github.com/aws/aws-app-mesh-controller-for-k8s/pkg/mesh"
 	"github.com/aws/aws-app-mesh-controller-for-k8s/pkg/virtualnode"
+	"github.com/aws/aws-app-mesh-controller-for-k8s/pkg/appmeshinject"
```

Comment:
nit, i'd suggest rename it to just be "inject" instead of "appmeshinject". but this works :D